client/src/components/DataFiles/DataFilesTable/DataFilesTable.js

```diff
@@ -23,13 +23,29 @@ const DataFilesTablePlaceholder = ({ section, data }) => {
   );
   const scheme = useSelector(state => state.files.params.FilesListing.scheme);
   const system = useSelector(state => state.pushKeys.target);
+  let currSystemHost = useSelector(state =>
+    state.systems.definitions.list.find(
+      sysDef => sysDef.id === state.files.params.FilesListing.system
+    )
+  );
+  currSystemHost = currSystemHost ? currSystemHost.storage.host : '';
   const loading = useSelector(state => state.files.loading[section]);
   const err = useSelector(state => state.files.error[section]);
   const modalRefs = useSelector(state => state.files.refs);
 
   const filesLength = data.length;
   const isGDrive = currentListing === 'googledrive';
 
+  useEffect(() => {
+    dispatch({ type: 'GET_SYSTEM_MONITOR' });
+  }, [dispatch]);
+  const downSysList = useSelector(state =>
```

Comment:
Consider using the same name for this `const` if both expressions of `state.systemMonitor` are the same in this file and in `AppForm.js`